firefly/firefly-drawer.js

```diff
@@ -155,22 +148,22 @@ Firefly.modules.drawer = function(FF) {
      */
     function resetPlayModel() {
         Firefly.model.runModel(modelSelect.value);
-        reset.innerText = '\u27F3';
+        //reset.innerText = '\u27F3';
```

Comment:
Left these commented out for now. Not sure if you wanted to go with Font Awesome since we'll be using a lot of icons. It's up to you, just let me know what you'd prefer and I can help adjust.

Reply:
Lmao I forgot I was using these -- yea these are safe to remove :D